iroh-net/src/magicsock/node_map/node_state.rs

```diff
@@ -287,20 +282,8 @@ impl NodeState {
                 (Some(best_addr.addr), self.relay_url())
             }
             best_addr::State::Empty => {
-                // No direct connection has been used before.  If we know of any possible
-                // candidate addresses, randomly try to use one while also sending via relay
-                // at the same time.
-                let addr = self
-                    .direct_addr_state
-                    .keys()
-                    .filter(|ipp| match ipp.ip() {
-                        IpAddr::V4(_) => true,
-                        IpAddr::V6(_) => have_ipv6,
-                    })
-                    .choose_stable(&mut rand::thread_rng())
-                    .map(|ipp| SocketAddr::from(*ipp));
-                trace!(udp_addr = ?addr, "best_addr is unset, use candidate addr and relay");
-                (addr, self.relay_url())
+                trace!("best_addr is unset, use relay");
```

Comment:
will we get here from somewhere above the node for which a relay only attempt will be made? Or should we have here `self.me` (or whatever the name of that field around here is)

Reply:
`2024-07-18T11:31:49.239813Z TRACE poll_send{me=xamcuaerdkwkwwjj}:get_send_addrs{node=dgjnkpaczxaekzxf}: iroh_net::magicsock::node_map::node_state: 285: best_addr is unset, use relay`

is how this shows up.  so i think this information is already suitably captured in the spans.